Library/Formula/xsd.rb

```diff
@@ -0,0 +1,97 @@
+class Xsd < Formula
+  desc "XML Data Binding for C++"
+  homepage "http://www.codesynthesis.com/products/xsd/"
+  url "http://www.codesynthesis.com/download/xsd/4.0/xsd-4.0.0+dep.tar.bz2"
+  version "4.0.0"
+  sha256 "eca52a9c8f52cdbe2ae4e364e4a909503493a0d51ea388fc6c9734565a859817"
+
+  depends_on "pkg-config" => :build
+  depends_on "xerces-c"
+
+  # Patches:
+  # 1. As of version 4.0.0, Clang fails to compile if the <iostream> header is
+  #    not explicitly included. The developers are aware of this problem, see:
+  #    http://www.codesynthesis.com/pipermail/xsd-users/2015-February/004522.html
+  # 2. As of version 4.0.0, building fails because this makefile invokes find
+  #    with action -printf, which GNU find supports but BSD find does not. There
+  #    is no place to file a bug report upstream other than the xsd-users mailing
+  #    list (xsd-users@codesynthesis.com). I have sent this patch there but have
+  #    received no response (yet).
+  patch :DATA
+
+  def install
+    ENV.append "LDFLAGS", `pkg-config --libs --static xerces-c`.chomp
+    system "make", "install", "install_prefix=#{prefix}"
+  end
+
+  def caveats; <<-EOS.undent
+    Note that this formula has been compiled with clang++ against libc++ because
+    xerces-c uses libc++ too. Apparently, this configuration is not well tested
+    and you might encounter some issues, see:
+    http://www.codesynthesis.com/pipermail/xsd-users/2015-February/004522.html
+    EOS
+  end
```

Comment:
It makes me a bit uncomfortable that we're basically telling users "stuff might go wrong". That seems to indicate this isn't ready for a Homebrew formula yet (particularly given it needs patched still to be installed)?